src/tests/end-to-end/common/element-identifiers/target-page-selectors.ts

```diff
@@ -6,3 +6,14 @@ export const TargetPageInjectedComponentSelectors = {
     issueDialog: '.insights-dialog-main-container',
     insightsVisualizationBox: '.insights-highlight-container',
 };
+
+export const TargetPageElementSelectors = {
+    targetPageNativeWidgetFirstRadio: '#input-radio-1',
+};
+
+export const tabStopShadowDomSelectors = {
```

Comment:
inconsistent casing between `const`s here

Reply:
removed the first one.

Reply:
...but this new one you've added is still inconsistent with the only other one in the file

Reply:
changed.